yoyodyne/train.py

```diff
@@ -317,9 +317,7 @@ def add_argparse_args(parser: argparse.ArgumentParser) -> None:
         help="Path to input validation data TSV.",
     )
     parser.add_argument(
-        "--train_from",
-        help="Path to ckpt checkpoint to resume training from. "
-        "Default: not enabled.",
+        "--train_from", help="Path to ckpt checkpoint to resume training from."
```

Comment:
Maybe remove "ckpt"?

Reply:
Done.

Reply:
Also rewrote slightly to avoid stigmatized particle stranding.